README.md

```diff
@@ -1,2 +1,126 @@
-# VMware.vROps
-Repository for VMware vRealize Operations
+# VMware vROps As Built Report
+
+# Getting Started
+Below are the instructions on how to install, configure and generate a VMware vROps As Built report.
+
+## Pre-requisites
+The following PowerShell modules are required for generating a VMware vROps As Built report.
+
+Each of these modules can be easily downloaded and installed via the PowerShell Gallery 
+
+- [AsBuiltReport Module](https://www.powershellgallery.com/packages/AsBuiltReport/)
+- [VMware PowerCLI Module](https://www.powershellgallery.com/packages/VMware.PowerCLI/)
+- [PowervROPs Module](https://github.com/ymmit85/PowervROps/)
+
+### Module Installation
+
+Open a Windows PowerShell terminal window and install each of the required modules as follows;
+```powershell
+install-module AsBuiltReport
+install-module VMware.PowerCLI
+import-module PowervROPs
```

Comment:
Additional info/steps required on how to install PowervROps

Reply:
Fixed